apis/apps/v1alpha1/clusterdefinition_types.go

```diff
@@ -1012,6 +1012,7 @@ type SwitchoverAction struct {
 	// Used to select the script that need to be referenced.
 	// When defined, the scripts defined in scriptSpecs can be referenced within the SwitchoverAction.CmdExecutorConfig.
 	//
+	// +kubebuilder:deprecatedversion:warning="This field is deprecated from KB 0.9.0"
```

Comment:
seems having no relation with `custom annotations`?

Reply:
> seems having no relation with `custom annotations`?

removed it